commissioning.tex

```diff
@@ -1,27 +1,48 @@
+The time from First Photon to System First Light is anticipated to be three months.
+During these three months, the focus will be on fine tuning the system including optical alignment and improving the image quality, collecting calibration data, and carrying out \textit{First Look} science programs. 
+
+For a detailed description of all the commissioning milestones and the most current dates, see \citeds{dmtn-232}.
+
 \subsection{Commissioning Schedule}
 \label{ssec:commissioning-schedule}
 
-In December 2022, as a result of reoptimizing the sequence of integration activities, the Rubin Construction Project decided to install the LSST Science Camera (LSSTCam) on the telescope earlier in the assembly sequence than previously planned. 
-As a consequence, no on-sky data will be taken with the Commissioning Camera (ComCam). 
-This change in strategy will produce more early-commissioning data than would have been available with ComCam on a similar timeframe. 
+Following the October 2023 project schedule workshop, the Rubin Construction Project is re-optimizing the sequence of integration activities in light of recent subcomponent delays, in particular, (i) delay in shipment of the Camera and (ii) necessary repairs to the  summit dome crane.
+If current estimates hold, it will be beneficial to re-implement on-sky data-taking with ComCam (previously removed from the schedule in order to install LSSTCam sooner).
+The updated plan calls for on-sky data to be taken with ComCam for approximately two months, around July-August 2024 to support Telescope commissioning ,primarily the Active Optics System. 
+This is approximately four months earlier than could be done with LSSTCam. 
+The Project will make the final, confirming decision about ComCam on-sky data-taking in February 2024.
 
 \begin{figure}[htb]
 \centering
-\includegraphics[width=0.95\linewidth]{figures/rubinobs_on-sky_commissioning_gantt.pdf}
-\caption{Detailed schedule of commissioning activities relative to System First Light, as of December 2022.}
-\label{fig:commissioning-gantt}
+\includegraphics[width=0.98\linewidth]{figures/rubinobs_on-sky_commissioning_and_early_science.pdf}
+\caption{Detailed schedule of commissioning  and early science activities relative to System First Light, as of October 2023.}
+\label{fig:commissioning-es-schedule}
+\vspace{0.1cm}
 \end{figure}
 
-Figure~\ref{fig:commissioning-gantt} shows the detailed schedule of commissioning activities relative to System First Light, as of December 2022.
-System First Light is currently expected between October 2024 and February 2025 (\S~\ref{sec:timeline}), about 3 months after First Photon. 
-The System First Light milestone marks the end of the the on-sky engineering phase and the start of the System Optimization and Science Validation phases. 
+Figure~\ref{fig:commissioning-es-schedule} shows the detailed schedule of commissioning and early science activities relative to System First Light, as of October 2023.
```

Comment:
The first block is inconsistent with ComCam. It is 6+months from ComCam on sky to FL. Or if this means FP with LSSTCam then that block is only 6 weeks. See dmtn232.